src/lib.rs

```diff
@@ -40,6 +41,7 @@ impl CrashReporter {
         organization: &str,
         application: &str,
         version: Option<String>,
+        distribution_group: Option<&str>,
```

Comment:
&str refers  to a string slice. It's similar to a pointer in C, and you can just point to a portion of the string.
The idea is that to use a pointer to the initial string and pass it around for usage, instead of an 'owned' string, which needs to be copied or cloned.

```
./lib.rs:162:            self.version.clone(),
./lib.rs:163:            self.distribution_group.clone(),

```